cli/awsbatch/awsbsub.py

```diff
@@ -127,6 +127,12 @@ def _get_parser():
         help="The number of nodes to reserve for the job. It enables Multi-Node Parallel submission",
         type=int,
     )
+    parser.add_argument(
+        "-g",
+        "--gpus",
+        help="The number of gpus to attach to the container. GPU jobs will only run on P family instances.",
```

Comment:
I'd write: _"only run on p2 and p3 instance families"_ according to the [documentation](https://docs.aws.amazon.com/en_us/batch/latest/userguide/batch-gpu-ami.html). What do you think?

Reply:
Or just mention the fact that these jobs run on a subset of instances and link to the batch docs